utils.py

```diff
@@ -487,7 +487,10 @@ def generate_random_string(length):
     Returns:
         str. Random string of specified length.
     """
-    return base64.urlsafe_b64encode(os.urandom(length))
+    random_string = ''
```

Comment:
Oh, hm, interesting. Was the old function creating strings that were too short? Based on reading the python docs, I'm not sure I understand why that would happen... what's an example?

Reply:
I wasn't sure about the case where we can have len(string) < length, but for me it was len(string) > length.

![image](https://user-images.githubusercontent.com/16653571/59081188-7a892f80-890a-11e9-8616-a2fca47c8d9e.png)

I read to docs in detail and I think it will only generate len(String) > length. I've made the changes!

Reply:
Oh, I see, urandom() creates a string of length 12 but urlsafe_b64encode might extend that. Thanks for clarifying!

We could alternatively just take the first "length" characters...

Reply:
Done!